config/client.go

```diff
@@ -18,6 +18,8 @@ func ParseClientConfig() (cfg ClientConfig) {
 	flag.StringVar(&cfg.ChainURL, "chain-url", "ws://127.0.0.1:8545", "'protocol://ip:port'")
 	flag.StringVar(&cfg.OpHost, "op-host", "127.0.0.1", "IP of hostname")
 	flag.IntVar(&cfg.OpPort, "op-port", 8401, "Account derivation index.")
+	flag.StringVar(&cfg.OpHost, "op-host", "127.0.0.1", "IP/host name of operator")
+	flag.IntVar(&cfg.OpPort, "op-port", 8401, "Port of operator.")
```

Comment:
The first commit adds there two duplicate lines but doesn't remove the old ones.